cpython-unix/build.py

```diff
@@ -52,6 +52,7 @@
 TARGETS_CONFIG = SUPPORT / "targets.yml"
 
 LINUX_ALLOW_SYSTEM_LIBRARIES = {
+    "atomic",
```

Comment:
We might need to make this conditional

See

- https://github.com/astral-sh/python-build-standalone/issues/410
- https://github.com/astral-sh/python-build-standalone/pull/411